app/controllers/api/services_controller.rb

```diff
@@ -54,28 +54,28 @@ def create
     creator = ServiceCreator.new(service: @service)
 
     if can_create? && creator.call(create_params)
-      flash[:notice] = t('flash.services.create.notice', resource_type: product_or_service_type)
+      flash[:notice] = t('flash.services.create.notice', resource_type: 'Product')
```

Comment:
If it's always "Product" now, do we still need `resource_type`?

Reply:
Good catch